packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -99,6 +99,36 @@ export class Select {
    */
   @Prop() emptyOptionListText = "No results found";
 
+  /**
+   * The <form> element to associate the select with. Use only this prop on searchable select.
```

Comment:
I think it's the other way around with the searchable select comment. Should be on all form props except form itself. Might also want rewording to something like "this prop will only be applied if searchable is true"